src/style.css

```diff
@@ -0,0 +1,117 @@
+body {
+    font-family: 'Inter', sans-serif;
+    background-color: #23252C;
+}
+
+.container {
+    margin: 44px auto;
+    width: 320px;
+    background-color: #1A1B21;
+    border-radius: 10px;
+}
+
+header {
+    display: flex;
+    text-align: center;
+    flex-direction: column;
+    align-items: center;
+}
+
+header>p {
+    margin-bottom: 15px;
+    color: #F5F5F5;
+}
+
+h1,
+h3,
+h4,
+p {
+    margin: 0;
+}
+
+.profile--image img {
+    height: 320px;
+    border-radius: 10px 10px 0 0;
+}
+
+header,
+main,
+.profile--buttons>button:nth-of-type(2) {
+    color: white;
+}
+
+.profile--title {
+    margin-top: 21px;
+    margin-bottom: 7px;
+    max-width: 300px;
+}
+
+.profile--job {
+    color: #F3BF99;
+    margin-bottom: 9px;
+}
+
+.profile--buttons {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    gap: 8px;
+    margin-bottom: 33px;
+}
+
+.profile--buttons>button {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 9px 13px 9px 11px;
+    gap: 8px;
+    box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.05);
+    border-radius: 6px;
+    box-sizing: border-box;
+    width: 115px;
+}
+
+.profile--buttons>button:nth-of-type(1) {
+    background-color: #FFFFFF;
+    border: 1px solid #D1D5DB;
+}
+
+.profile--buttons>button:nth-of-type(2) {
```

Comment:
To wygląda na sytuację w której warto by zrobić z tego po prostu varianty buttona i przypisać do klas, a nie stylować za pomocą pseudoklas

np

.button--danger {
  ...specyficzne style dla czerwonego przycisku
}

Generalnie stylowanie na podstawie pozycji w DOM jest dość ryzykowne i podatne na błędy w przyszłości